DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,13 @@ public function getConfigTreeBuilder()
         $rootNode->setBuilder(new AllowExtraPropertiesNodeBuilder());
 
         $rootNode->children()
+            ->arrayNode('allowedParamConverters')
+                ->prototype('array')
+                    ->children()
+                        ->scalarNode('name')->isRequired()->end()
```

Comment:
Because `foo_param_converter: ~` looks strange too me. But if you have a better solution in mind I'm glad to change it :)